src/basic/generate-af-list.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+cpp -dM -include sys/socket.h - </dev/null | \
```

Comment:
Or use `$CC -E` because I don't think it's currently possible to configure a cross cpp with meson.